src/art/model.py

```diff
@@ -52,6 +100,38 @@ async def register(self, api: "LocalAPI") -> None:
         self._api = api
         await self._api.register(self)
 
+    def openai_client(
+        self,
+    ) -> AsyncOpenAI:
+        if self.inference_api_key is None or self.inference_base_url is None:
+            raise ValueError(
+                "OpenAI client not yet available. You must call `model.register()` first."
```

Comment:
I don't believe this error message makes sense anymore, since comparison models' inference values aren't populated by the register command.

Maybe we should move this message to an overloaded function on `TrainableModel` that checks for an api key and base url and throws this error if they're not present, else calls super?

Reply:
👍 updated to have different error messages for `Model` vs `TrainableModel`.